File: Array/reverseArray.java
// Program to reverse an array of integers

package Array;

public class reverseArray {

    static void reverseArr(int[] arr, int start, int end) {

        int temp;

        while (start < end) {
            temp = arr[start];
            arr[start] = arr[end];
            arr[end] = temp;
            start++;
            end--;
        }
    }

    static void printArray(int[] arr, int size) {
        for (int i = 0; i < size; i++) {
            System.out.print(arr[i] + " ");
        }
        System.out.println();
    }

    public static void main(String[] args) {

        int[] arr = {1,2,3,4,5,6};
        System.out.print("Entered array is: ");
        printArray(arr, 6);
        reverseArr(arr, 0, 5);
        System.out.print("Reversed array is: ");
        printArray(arr, 6);
    }
}

/* 
Output: 

Entered string is: 1 2 3 4 5 6
Reversed string is: 6 5 4 3 2 1
*/

File: Array/reverseArray.py
#Program to reverse an array

def reverseArray(arr):
    print(arr[::-1])


print("Enter length of your array: ",end="")
len = int(input())
arr = []
print("Enter your array:")
for i in range(len):
    inp = input()
    arr.append(inp)


print("Entered array is ",end="")
print(arr)
print("Reversed array is ",end="")
reverseArray(arr)


"""
Output:

Enter length of your array: 6
Enter your array:
123
789
abc
xyz
???
1b4
Entered array is ['123', '789', 'abc', 'xyz', '???', '1b4']
Reversed array is ['1b4', '???', 'xyz', 'abc', '789', '123']
"""